contents/docs/experiments/common-questions.mdx

```diff
@@ -89,3 +89,14 @@ You can check your current exposure count on the experiment results page. If you
 - Verify your feature flag implementation.
 - Consider increasing your feature flag's rollout percentage.
 - Ensure your exposure events are firing correctly.
+
+## Why do my experiment's exposure counts not match insights?
+
+Currently, insights aren't able to account for users that have been exposed to `$multiple` variants of your feature flag. We are actively working on improving this, however in the meantime exposures as measured by the experiment metrics are more accurate.
```

Comment:
Could you give an example of what type of insight this would happen to? I can't really think of any easily. 

```suggestion
Currently, insights aren't able to account for users that have been exposed to `$multiple` variants of your feature flag. We are actively working on improving this. In the meantime, exposures as measured by the experiment metrics are more accurate.
```